int64 num1
int64 num2
string name
---
string name_id
int64 sum